packages/angular/projects/clr-angular/src/utils/variables/_variables.global.scss

```diff
@@ -265,13 +265,13 @@ $clr-layers: (
   datagrid-header: 501,
   datagrid-header-sticky: 502,
   datagrid-host: 590,
-  column-switch: 600,
   dropdown-menu: 1000,
```

Comment:
I think https://github.com/vmware/clarity/issues/4338 can be fixed updating this line, can you check as part of this?

Reply:
@gnomeontherun I forgot about that one, thanks for reminding me. Will update PR to address #4338 as well